src/gui/gui_msgs.h

```diff
@@ -35,50 +35,50 @@ There is NO WARRANTY, to the extent permitted by law.
 constexpr char help_msg[] =
         R"(Usage: dosbox [OPTION]... [FILE]
 
-These are common options:
+List of common options:
 
-  -h, --help          Displays this message.
+  --printconf          Print the location of the default configuration file.
 
-  --printconf         Prints the location of the default configuration file.
+  --editconf           Open the default configuration file in a text editor.
 
-  --editconf          Open the default configuration file in a text editor.
+  -c <command>         Run the specified DOS command before running FILE.
+                       Multiple commands can be specified.
 
-  -c <command>        Runs the specified DOS command before running FILE.
-                      Multiple commands can be specified.
+  -noautoexec          Don't perform any [autoexec] actions.
 
-  -noautoexec         Don't perform any [autoexec] actions.
+  -noprimaryconf       Don't read settings from the primary configuration file
+                       located in your user folder.
 
-  -noprimaryconf      Don't read settings from the primary configuration file
-                      located in ~/.config/dosbox/dosbox-staging.conf.
+  -nolocalconf         Don't read settings from "dosbox.conf" if present in
+                       the current working directory.
 
-  -nolocalconf        Don't read settings from "dosbox.conf" if present in
-                      the local current working directory.
+  -conf <configfile>   Start with the options specified in <configfile>.
+                       Multiple configfiles can be specified.
 
-  -conf <configfile>  Start dosbox with the options specified in <configfile>.
-                      Multiple configfiles can be present at the commandline.
+  --working-dir <path> Set working directory to <path>. DOSBox will act as if
+                       started from this directory.
 
```

Comment:
Really liking the improved ordering, @johnnovak.

**First up** - we start with ways to learn about conf-file settings, because the best way to run games is using game-specific conf files, versus manually entering CLI launch flags and specifying runtime settings, is tedious and prone to inconsistencies- both which will discourage users.

Reply:
Yes I think so too; glad you noticed it!